### ISSUE TITLE
thundersvmScikit.SVC `verbose=False` doesn't seem to work

### ISSUE THREAD

**Issue by bkj** [zeyiwen/thundersvm#59]
Running
```
clf   = SVC(kernel=2, n_cores=30, max_iter=20, verbose=False).fit(X, y)
```
logs
```
2018-05-02 22:38:14,898 INFO [default] #instances = 71829, #features = 2
2018-05-02 22:38:14,905 INFO [default] #classes = 2
2018-05-02 22:38:14,925 INFO [default] working set size = 1024
2018-05-02 22:38:14,929 INFO [default] training start
2018-05-02 22:38:14,938 INFO [default] global iter = 0, total local iter = 74, diff = 2
```

Is there some way to turn off the logging?

Thanks
~ Ben

**Comment by bkj** [comment]
Quick fix -- the `verbose` flags were flipped.  Fixed in pull request #60

**Comment by zeyiwen** [comment]
Thank you for the contributions! We have merged the pull requests.
